java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -5,7 +5,8 @@
 import java.util.concurrent.CompletableFuture;
 
 /**
- * Generic Commands interface to handle generic commands for all server requests.
+ * Supports commands and transactions for the "Generic Commands" group for standalone clients and
```

Comment:
Actually, not
[Generic commands group](https://redis.io/commands/?alpha=t&group=generic) contain `touch`, `ttl` and `type` only.
We need to rename this to something else.